tests/framework/prototype.py

```diff
@@ -146,13 +146,19 @@ def check_pod_running(pods) -> bool:
             return False
     return True
 
-def shell_subprocess_run(command, check = True):
+def shell_subprocess_run(command, cr_namespace = None, check = True):
```

Comment:
This framework is possible to use for other operators, and thus maybe we should not couple them together.

Reply:
I agree -- K8s concepts shouldn't appear in a command utility.